lib/utils/compiler.js

```diff
@@ -112,7 +112,11 @@ function compilerCallback(compiler, err, stats, lastHash, options, outputOptions
     }
 }
 
-module.exports = async function webpackInstance(opts, shouldUseMem) {
+async function getCompiler(opts) {
```

Comment:
This function doesn't need to be `async`, it can just return the unhandled promise. Same goes for the subsequent usage of `await`.

Reply:
Are you sure the subsequent `await` is not needed? Is there something coming in webpack 5 where it might return a promise when `webpack()` is called? I assume that's why the `await` was there originally.